vtex/omnishipping
# CHK-482: Fix check for number field validity

#### What is the purpose of this pull request?

Update the checks for the number validity to use `address.number.valid`.

#### What problem is this solving?

Fix the checks for the number validity, that where previously using the `address.number.value` instead of `address.number.valid`, which caused issues on countries that do not have number in the addresses.

#### How should this be manually tested?

[Workspace](https://lucas--vtexgame1geo.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&sc=2).

You can check this works by using the "United Emirate Arabes" country in the workspace above and with the address "W Corniche Road" (you can select the ~first~ only option). You should be able to proceed to the payment step without issue.

#### Screenshots or example usage

#### Types of changes
